### ISSUE TITLE
terraform crash during plan

### ISSUE THREAD

**Issue by eugenetaranov** [hashicorp/terraform#4019]
Hello,

just tried to tun terraform pull and it crashed. Crash log attached.
Could you please look ?

[crash.txt](https://github.com/hashicorp/terraform/files/41457/crash.txt)
v0.6.6, OSX

Thanks,
Eugene

**Comment by eugenetaranov** [comment]
quick update, might be important: it timed out when I was running it last time during aurora cluster spinup. Maybe this broke tfstate files.

**Comment by radeksimko** [comment]
Cutting out the interesting part of the crash log:

```
2015/11/23 12:11:44 [DEBUG] terraform-provider-aws: panic: runtime error: invalid memory address or nil pointer dereference
2015/11/23 12:11:44 [DEBUG] terraform-provider-aws: [signal 0xb code=0x1 addr=0x38 pc=0x122521]
2015/11/23 12:11:44 [DEBUG] terraform-provider-aws: 
2015/11/23 12:11:44 [DEBUG] terraform-provider-aws: goroutine 65 [running]:
2015/11/23 12:11:44 [DEBUG] terraform-provider-aws: github.com/hashicorp/terraform/builtin/providers/aws.resourceAwsRDSClusterInstanceRead(0xc2083e3e00, 0x7c65c0, 0xc2080be840, 0x0, 0x0)
2015/11/23 12:11:44 [DEBUG] terraform-provider-aws: 	/opt/gopath/src/github.com/hashicorp/terraform/builtin/providers/aws/resource_aws_rds_cluster_instance.go:135 +0x321
```

**Comment by jen20** [comment]
Hi @eugenetaranov! Thanks for reporting this crash. Could you confirm the command you were executing when you saw this crash? From looking at the code paths and discussing with @phinze, it doesn't seem like a simple `terraform remote pull` could cause execution to follow this path as it doesn't refresh.

**Comment by eugenetaranov** [comment]
Hi,
you are correct, I think I made a typo, sorry for that. Command was "~/bin/terraform/terraform plan"

**Comment by catsby** [comment]
Hey @eugenetaranov – was the Cluster Instance renamed, or are there multiple instances with a similar name by any chance?

**Comment by eugenetaranov** [comment]
Nope, it was not renamed afaik, but there was aws_rds_cluster_instance resource with count set to 2

**Comment by catsby** [comment]
If you have the configuration that would be helpful, omitting any secrets. I'll continue to dig in as well, thanks!

**Comment by eugenetaranov** [comment]
edited last reply, please see if it is enough

**Comment by phinze** [comment]
Hi @eugenetaranov - this crash should be fixed with https://github.com/hashicorp/terraform/pull/5717 which landed today and will be included in the 0.6.14 release slated for Monday. :+1:
